.haml.yml

```diff
@@ -33,7 +33,7 @@ linters:
 
   LineLength:
     enabled: true
-    max: 80
+    max: 200
```

Comment:
This is too long for my tastes, but not going to slow things down based on that :-)